web/client/utils/LayersUtils.js

```diff
@@ -153,17 +153,19 @@ export const getNestedGroupTitle = (id, groups = []) => {
 
 /**
  * Flatten nested array to a one-level array
- * @param {object[]} array of objects
- * @returns {object[]} flattened array
+ * @param {Object[]} array of objects
+ * @returns {Object[]} flattened array
  */
-export const flattenArrayOfObjects = (array) => {
+export const flattenArrayOfObjects = (arrayOfObj) => {
```

Comment:
what if we use this https://lodash.com/docs/4.17.15#flattenDeep ?

Reply:
`_.flattenDeep` seems to be expecting an array but in some cases what is passed is just a single object